### PR TITLE
:bug: input not applying prop 'onPaste'

### PR DESCRIPTION
**Pull request by vmenard**

### CODE REVIEW

File: src/input/field/field.jsx [hunk 1]
```diff
@@ -4,7 +4,9 @@ import { onlyUpdateForPropTypes } from 'recompose'
 import classnames from 'classnames'
 import styles from '../../../src/input/input.styles.scss'
 
-const Field = ({ input, placeholder, type, meta: { touched, error, submitFailed }, ...rest }) => {
+const Field = ({
```
Comment (fabienjuif) [inline src/input/field/field.jsx]: I think the fix is more about removing `onlyUpdateForPropTypes` since we use `...rest`.
Can you try like that ?

Comment (fabienjuif) [reply src/input/field/field.jsx]: You should let the `input` fix as it is.
But removing `onlyUpdateForPropTypes` just here

Comment (vmenard) [reply src/input/field/field.jsx]: I already try this, but it doesn't work

Comment (fabienjuif) [reply src/input/field/field.jsx]: Did you try to 
 - let Input fix you did (declare onPaste on **input**)
 - remove the `onlyUpdateForPropTypes` on **field**

(both modifications ?)

Comment (vmenard) [reply src/input/field/field.jsx]: Sorry, I didn't understand :) this solution works

**Comment by fabienjuif** [comment]
@vmenard do you have other bugs on pipe ? Or should I publish asap ?

**Comment by vmenard** [comment]
No, you can publish thanks

**Comment by fabienjuif** [comment]
done :)

**Pull request merged**